pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -261,6 +269,17 @@ func desiredLoadBalancerService(ci *operatorv1.IngressController, deploymentRef
 			} else {
 				service.Annotations[awsLBHealthCheckIntervalAnnotation] = awsLBHealthCheckIntervalDefault
 			}
+
+			if platform != nil && platform.AWS != nil && len(platform.AWS.ResourceTags) > 0 {
```

Comment:
```suggestion
			if platform.AWS != nil && len(platform.AWS.ResourceTags) > 0 {
```

The `platform != nil` check here is redundant, right?

Reply:
Yes, we're guarded by the block "above". Removed.

Reply:
Done in https://github.com/openshift/cluster-ingress-operator/pull/578/commits/154643c1444bbbe4c7e68675680cc397b59d8f3e.